coremltools/converters/xgboost/_tree_ensemble.py

```diff
@@ -278,3 +282,62 @@ def convert_tree_ensemble(
         )
 
     return mlkit_tree.spec
+
+
+def _booster_feature_names_workaround(
+        model: _xgboost.Booster or _xgboost.XGBRegressor or _xgboost.XGBClassifier, feature_names):
+    """
+    Removes booster feature names. This is intended as a workaround for faulty JSON dump
+    generated by get_dump(dump_format='json') and dumo_model(..., dump_format='json')
```

Comment:
"dumo_model" should be another "dump_model"

Reply:
thank you for pointing this out - I'll correct it